src/config/couchdb.rst

```diff
@@ -179,10 +179,13 @@ Base CouchDB Options
         * ``nolb``: The /_up endpoint will return a 404 response.
         * ``false``: The node responds normally, /_up returns a 200 response.
 
+        .. highlight:: none
+
         It is expected that the administrator has configured a load balancer
         in front of the CouchDB nodes in the cluster. This load balancer should
         use the /_up endpoint to determine whether or not to send HTTP requests
-        to any particular node. For HAProxy, the following config is appropriate ::
+        to any particular node. For HAProxy, the following config is
+        appropriate::
```

Comment:
I think you better replace `::` with `.. code-block:: none` definition. That's more error-prone solution. Same for the rest similar cases.